pkgs/development/python-modules/myst-parser/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, flit-core
+, docutils
+, mdit-py-plugins
+, pyyaml
+, sphinx
+, typing-extensions
+  # , beautifulsoup4
+  # , coverage
+  # , pytest-cov
+  # , pytest-param-files
+  # , pytest-regressions
+  # , sphinx-pytest
+  # , pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "myst-parser";
+  version = "0.18.0";
+
+  src = fetchFromGitHub {
+    owner = "executablebooks";
+    repo = pname;
+    rev = "v" + version;
+    sha256 = "sha256-GEtrC7o5YnkuvBfQQfhG5P74QMiHz63Fdh1cC/r5CF0=";
+  };
+
+  format = "flit";
+
+  nativeBuildInputs = [ flit-core ];
+
+  propagatedBuildInputs = [
+    docutils
+    mdit-py-plugins
+    pyyaml
+    sphinx
+    typing-extensions
+  ];
+
+  pythonImportsCheck = [ "myst_parser" ];
+
+  # Tests are actually failing
+  doCheck = false;
+
+  # checkInputs = [
+  #   beautifulsoup4
+  #   coverage
+  #   pytest-cov
```

Comment:
We don't care much about coverage. Usually we patch those requirements out and skip everything coverage-related.

Here it looks like that coverage is not triggered by `pytest`. Remove `coverage` and `pytest-cov`.